_plugins/server_generator.rb

```diff
@@ -1,5 +1,7 @@
 module Jekyll
   class ServerPage < Page
+    include Jekyll::ThanksToFilter
```

Comment:
I've refactored slightly in abf8ab5 which means the `include` isn't needed any more. Does that look better?